src/wpool_process.erl

```diff
@@ -42,8 +42,9 @@
 -spec start_link(wpool:name(), module(), term(), [wpool:option()]) ->
         {ok, pid()} | ignore | {error, {already_started, pid()} | term()}.
 start_link(Name, Module, InitArgs, Options) ->
+  Worker_opt = proplists:get_value(worker_opt, Options, []),
```

Comment:
It would be good to remove the option from the list, i.e. using `lists:keytake` instead of `proplists:get_value`